rails6-example/.database_consistency.yml

```diff
@@ -2,6 +2,14 @@ DatabaseConsistencySettings:
   color: true
   log_level: DEBUG
 
+# Ignore false positive from Rails' ActionText and ActiveStorage
```

Comment:
Hey, didn't we just merge this already? https://github.com/djezzzl/database_consistency/pull/54/commits/4a3c4f999db20da1f6a7de256620cf8aedc4fe91

That's strange we don't have any conflicts because this PR has different commit hash.

Reply:
That's because my branch included the other commit. Sorry I missed that.

Reply:
It's still here. Could you please rebase on top of master and remove wrong commits?